Podcast/FollowerFollowingViewController.swift

```diff
@@ -0,0 +1,110 @@
+    var usersTableView: UITableView!
+    var loadingActivityIndicator: NVActivityIndicatorView!
+    var refreshControl: UIRefreshControl!
+    
+    var users: [User] = []
+    var followersOrFollowings: UserFollowsType!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .podcastWhiteDark
+        title = followersOrFollowings == .Followers ? "Followers" : "Following"
+
+        guard let appDelegate = UIApplication.shared.delegate as? AppDelegate else { return }
+        // Do any additional setup after loading the view.
+        usersTableView = UITableView(frame: CGRect.zero)
+        usersTableView.contentInset.bottom = appDelegate.tabBarController.tabBarHeight
+        usersTableView.delegate = self
+        usersTableView.dataSource = self
+        usersTableView.backgroundColor = .clear
+        usersTableView.separatorStyle = .none
+        usersTableView.showsVerticalScrollIndicator = false
+        usersTableView.register(SearchPeopleTableViewCell.self, forCellReuseIdentifier: cellIdentifier)
+        usersTableView.reloadData()
+        view.addSubview(usersTableView)
+        
+        loadingActivityIndicator = createLoadingAnimationView()
+        loadingActivityIndicator.center = view.center
+        view.addSubview(loadingActivityIndicator)
+        
+        refreshControl = UIRefreshControl()
+        refreshControl.tintColor = .podcastTeal
+        refreshControl.addTarget(self, action: #selector(FollowerFollowingViewController.handleRefresh), for: UIControlEvents.valueChanged)
+        usersTableView.addSubview(refreshControl)
+        
+        usersTableView.snp.makeConstraints { make in
+            make.top.equalToSuperview()
+            make.leading.equalToSuperview()
+            make.trailing.equalToSuperview()
+            make.bottom.equalToSuperview()
+        }
+        
+        fetchUsers()
+    }
+    
+    @objc func handleRefresh() {
+        fetchUsers()
+    }
+    
+    func fetchUsers() {
+        let endpointRequest = FetchUserFollowsByIDRequest(userId: System.currentUser!.id, type: followersOrFollowings)
+        endpointRequest.success = { request in
+            guard let follows = request.processedResponseValue as? [User] else { return }
+            self.users = follows
+            self.refreshControl.endRefreshing()
+            self.loadingActivityIndicator.stopAnimating()
+            self.usersTableView.reloadSections([0] , with: .automatic)
+        }
+        System.endpointRequestQueue.addOperation(endpointRequest)
+    }
+    
+    //
+    // MARK: TableView Delegate & DataSource Methods
+    //
+
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return users.count
+    }
+    
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        // TODO: Bring up user profile
```

Comment:
Add this to this PR

Reply:
If we have endpoint which we should

Reply:
I didn't want to add yet, that's the next part is fixing more of profile. It's still partially broken I believe.

Reply:
ok!